# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gettext/hotfix-clang.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gettext-0.21/gettext-tools/libgettextpo/obstack.c.vanilla	2023-05-12 17:19:16.262534655 +0200
+++ gettext-0.21/gettext-tools/libgettextpo/obstack.c	2023-05-12 17:19:25.458535319 +0200
@@ -326,7 +326,7 @@
 #   include <libio/iolibio.h>
 #  endif
 
-static _Noreturn void
+static __attribute_noreturn__ void
 print_and_abort (void)
 {
   /* Don't change any of these strings.  Yes, it would be possible to add
--- gettext-0.21/gettext-tools/gnulib-lib/obstack.c.vanilla	2023-05-12 17:09:12.263491044 +0200
+++ gettext-0.21/gettext-tools/gnulib-lib/obstack.c	2023-05-12 17:18:21.602530709 +0200
@@ -326,7 +326,7 @@
 #   include <libio/iolibio.h>
 #  endif
 
-static _Noreturn void
+static __attribute_noreturn__ void
 print_and_abort (void)
 {
   /* Don't change any of these strings.  Yes, it would be possible to add
--- gettext-0.21/libtextstyle/lib/obstack.c.vanilla	2023-05-12 17:10:58.013498679 +0200
+++ gettext-0.21/libtextstyle/lib/obstack.c	2023-05-12 17:16:35.564523052 +0200
@@ -326,7 +326,7 @@
 #   include <libio/iolibio.h>
 #  endif
 
-static _Noreturn void
+static __attribute_noreturn__ void
 print_and_abort (void)
 {
   /* Don't change any of these strings.  Yes, it would be possible to add
